spec/features/dataset_show_page_spec.rb

```diff
@@ -10,18 +10,27 @@
 
   feature 'Meta data' do
     before(:each) do
-      dataset = DatasetBuilder.new.build
+      @dataset = DatasetBuilder.new.build
 
-      index_and_visit(dataset)
+      index_and_visit(@dataset)
```

Comment:
Does this need to an instance variable? It's not referenced in the new scenario.

Reply:
Good spot - thanks